ui/shared/components/table/DataTable.jsx

```diff
@@ -68,6 +68,7 @@ class DataTable extends React.PureComponent {
     downloadAlign: PropTypes.string,
     loadingProps: PropTypes.object,
     filterContainer: PropTypes.object,
+    onClick: PropTypes.func,
```

Comment:
is this used?

Reply:
No. I thought we could show the IGV when the user clicks on a row.